### DISCUSSION TITLE
Filter existing Links

### DISCUSSION THREAD

**Discussion by CathyWilms** [SkepticMystic/graph-analysis#70] [Ideas]
Hi,
it would be very usefull if you could filter existing links in the current note out of the analysis. That way you could use it as a "considered to link this?" (specially with jira) without having to check against existing links all the time.

**Comment by SkepticMystic** [comment]
Hey @CathyWilms 
I've done a bit of testing on this, and I think there's a solution. If you add the following CSS to your vault, it should hide all rows to which the current note is linked:
```css
tr.GA-linked {
  display: none;
}
```

I've not touched the code in ages, so I haven't actually been able to test this "for real". Please let me know if that works. But it should theoretically be possible, since the `GA-linked` class is added to rows that are linked.
